web/src/beta/features/Visualizer/Crust/Infobox/Block/builtin/Markdown/index.tsx

```diff
@@ -71,3 +75,12 @@ const Wrapper = styled("div")(() => ({
     height: "auto"
   }
 }));
+
+const MarkdownWrapper = styled("div")(() => ({
+  "& > ul": {
+    listStyleType: "initial"
+  },
+  "& > ol": {
+    listStyleType: "-moz-initial"
```

Comment:
```suggestion
    listStyleType: "initial"
```
Could we use this directly?

Reply:
I think we can use `decimal`.